frontend/device/remarkable/powerd.lua

```diff
@@ -0,0 +1,32 @@
+local BasePowerD = require("device/generic/powerd")
+
+-- TODO onlder firmware doesn't have the -0 on the end of the file path
+local base_path = '/sys/class/power_supply/bq27441-0/'
+
+local Remarkable_PowerD = BasePowerD:new{
+    is_charging = nil,
+    fl_min = 0,
```

Comment:
fl_min and fl_max can be removed on devices without hasFrontlight

Reply:
OK, removed